inject-java/src/test/groovy/io/micronaut/inject/annotation/AnnotationMapperSpec.groovy

```diff
@@ -113,10 +145,31 @@ class Test {
         }
     }
 
+    static class  CustomAroundMapper implements AnnotationMapper<CustomAround> {
+        @Override
+        List<AnnotationValue<?>> map(AnnotationValue<CustomAround> annotation, VisitorContext visitorContext) {
+
+            AnnotationValueBuilder<CustomAround> customAroundValueBuilder =
+                AnnotationValue.builder(CustomAround.class);
+
+            AnnotationValueBuilder<Type> typeAnnotationValueBuilder =
+                AnnotationValue.builder(Type.class);
+
+            AnnotationValueBuilder<Around> aroundAnnotationValueBuilder =
+                AnnotationValue.builder(Around.class)
+                    .stereotype(typeAnnotationValueBuilder.build(), customAroundValueBuilder.build());
```

Comment:
Type should not be a stereotype of Around, Type and Around should both be stereotypes of CustomAround. The type should also reference a class. It would be good to build a context with the runtime compiled class and verify the advice is actually referenced